dotnet/frontend/appsettings.json

```diff
@@ -8,9 +8,6 @@
   },
   "AllowedHosts": "*",
   "Honeycomb": {
-    "ServiceName": "frontend-dotnet",
-    "Endpoint": "https://api-dogfood.honeycomb.io",
-    "ApiKey": "",
-    "Dataset": "greetings"

```

Comment:
Losing `Dataset` does seem a little strange. This definitely gets into "we're in tilt/docker territory" and diverges from our docs.

Is that fine?

Reply:
The dataset should not be set for each Service  -- it's the other global that you need to set.

To counter your question -- what should happen if `HONEYCOMB_DATASET` and  this `Dataset` disagree?

Reply:
(Noting of course that this is all pre-E&S transition)

Reply:
>To counter your question -- what should happen if HONEYCOMB_DATASET and this Dataset disagree?

In this case, the environment variable will win.

Sounds good overall!

Reply:
I see your point Phillip. I think we should either:
- delete the entire honeycomb/otlp section from the configs and embrace that we're in tilt world
- keep the config (with all the values) as an example, understanding that env vars override it

Reply:
I'm a big fan of keeping this all runnable and understandable without Tilt, even if it means some env vars are just commented out. We intend to keep using this as an up-to-date example repo to point people to for multiple services, mixed instrumentations, etc... so it's probably best to ensure it's very clear where/how these variables can be set.

Reply:
I agree with @JamieDanielson, someone just want to refer to our example as code example and having all the settings 'hidden' in the tilt file can be confusing.

Reply:
got it. Adding them back in. 

```json
    "ServiceName": "frontend-dotnet",
    "Endpoint": "https://api.honeycomb.io",
    "ApiKey": "",
    "Dataset": "greetings"
```